en/middleware/micrortps_manual_code_generation.md

```diff
@@ -100,7 +81,7 @@ void deserialize_sensor_combined(struct sensor_combined_s *output, char *input,
 
 ### RTPS message IDL files
 
-IDL files are generated from the uORB **.msg** files ([for supported uORB topics](#supported-uorb-messages)). These can be found in: **build\__BUILDPLATFORM_/src/modules/micrortps_bridge/idl/**
+IDL files are generated from the uORB **.msg** files for selected uORB topics in the generation of the bridge. These can be found in: **src/modules/micrortps_bridge/micrortps_agent/idl/**
```

Comment:
There are two ways that depend on you generate manually or automatically. Manually by the script and --send and --receive options, automaticlly through the .cmake files.